integration-tests/src/test/resources/tunneling/JmsTestClient.java

```diff
@@ -0,0 +1,122 @@
+import java.util.Hashtable;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+
+import javax.jms.Destination;
+import javax.jms.ConnectionFactory;
+import javax.jms.Connection;
+import javax.jms.Session;
+import javax.jms.Message;
+import javax.jms.DeliveryMode;
+import javax.jms.TextMessage;
+import javax.jms.MessageProducer;
+import javax.jms.MessageConsumer;
+import javax.jms.JMSException;
+
+import javax.jms.JMSContext;
+import javax.jms.JMSConsumer;
+import javax.jms.JMSProducer;
+import javax.jms.JMSRuntimeException;
+
+public class JmsTestClient {
+
+  public  String username ="weblogic";
+  public  String password ="welcome1";
+
+  public  String clusterurl ="t3://localhost:7001";
+  public  String testQueue ="jms/DistributedQueue";
+  public  String testcf ="jms.ClusterConnectionFactory";
+
+  public JmsTestClient(String[] args)

```

Comment:
add javadoc with purpose of this class

Reply:
/**
 * This JMS client that sends 300 messages to a Uniform Distributed Queue 
 * using load balancer http(s) url which maps to custom channel on cluster 
 * member server on WebLogic cluster.
 * It also verifies that the messages are load balanced across 2 member.
 * The test returns success(0) if it finds 150 messages on each member 
 * else returns failure (-1)  
 * Usage java JmsTestClient http(s)://host:port
 */